language/diem-framework/releases/artifacts/current/transaction_script_builder.rs

```diff
@@ -3265,6 +3265,50 @@ impl ScriptCall {
             None => None,
         }
     }
+
```

Comment:
IIRC this is a generated file so it might not be the best place for this. Maybe this could be added to the generator? or done in the transaction-builder crate itself?

Reply:
the change in `language/transaction-builder/generator/src/rust.rs` adds the codegen that generates this function : )

Reply:
lol missed that :P